docs/source/api/logprob.rst

```diff
@@ -21,3 +21,51 @@ Conditional probability
 
    conditional_logp
    transformed_conditional_logp
+
+
+Derived probability functions
+-----------------------------
+
+In PyMC, users can create their own custom distributions, which refers to the ability to define and utilize probability distributions not found in the standard set provided.
+While PyMC provides a wide range of common probability distributions (e.g., Normal, Bernoulli, etc.), there may be cases where you need to use a distribution that is not available by default. In such cases, you can create your own custom distribution using the :class:`~pm.CustomDist` class provided by PyMC.
```

Comment:
```suggestion
While PyMC provides a wide range of common probability distributions (e.g., Normal, Bernoulli, etc.), there may be cases where you need to use a distribution that is not available by default. In such cases, you can create your own custom distribution using the :class:`~pymc.CustomDist` class provided by PyMC.
```

The "name" used for the hyperlink should be the whole **import path** where it is documented (not where it is found in the codebase). `pm` is therefore not available. To shorten it, the `~` is available, so the text only shows the object and skips the library and modules (if any).